src/approaches/random_options_approach.py

```diff
@@ -0,0 +1,43 @@
+"""An approach that just executes random options.
+"""
+
+from collections import defaultdict
+from typing import Callable, List
+from predicators.src.approaches import BaseApproach
+from predicators.src.structs import State, Task, Action
+from predicators.src.settings import CFG
+from predicators.src import utils
+
+
+class RandomOptionsApproach(BaseApproach):
+    """Samples random options (and random parameters for those options).
+    """
+    @property
+    def is_learning_based(self) -> bool:
+        return False
+
+    def _solve(self, task: Task, timeout: int) -> Callable[[State], Action]:
+        options = sorted(self._initial_options, key=lambda o: o.name)
+        plan: List[Action] = []
```

Comment:
this is okay because the env is deterministic etc, but it's a little odd -- why do we need to plan at all if we're executing random options?